.github/workflows/CI.yml

```diff
@@ -35,7 +35,9 @@ jobs:
           path: ${{ steps.yarn-cache.outputs.dir }}
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
       - name: Yarn install
-        run: yarn --frozen-lockfile
+        run: |
+          yarn add -D comsync
```

Comment:
lol no i meant add it in advance in the repo, not in GHA

Reply:
Doesn't that go against the point of adding it as a peer dependency? Or do we have it listed as both a dev and a peer dependency then?

Reply:
Yes, both dev and peer. That's what I did in the comsync repo. Since it's not an actual dependency, it's still not included in the bundle and other users still have to install it themselves.